### PR TITLE
Moved 8m surefire config for page cache to root pom.xml

### PR DESCRIPTION
**Pull request by tinwelint**
so that it will apply to all testing. Tests that want bigger page cache
will have to explicitly specify that.

### CODE REVIEW

**Comment by chrisvest** [comment]
@tinwelint I think we can safely lower the lower-bound on the `GraphDatabaseSettingsTest.mustHaveReasonableDefaultPageCacheMemorySizeInBytes` to make that one pass.

I don't know what's wrong with the other test. I think it should be investigated.

**Comment by tinwelint** [comment]
@chrisvest made that change. The other test was actually a real issue, which is now fixed as part of this PR, see b6d92e25c30e32aff92c0521b2c4da52f109a635

File: community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/ProducerStep.java [hunk 1]
```diff
@@ -56,6 +56,7 @@ public void run()
                 }
                 catch ( Throwable e )
                 {
+                    e.printStackTrace();
```
Comment (chrisvest) [inline community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/ProducerStep.java]: Are you sure?

File: community/kernel/src/test/java/org/neo4j/unsafe/impl/batchimport/RelationshipGroupDefragmenterTest.java [hunk 2]
```diff
@@ -213,6 +213,8 @@ private void verifyGroupsAreSequentiallyOrderedByNode()
                 continue;
             }
 
+//            System.out.println( "read:" + groupRecord );
+
```
Comment (chrisvest) [inline community/kernel/src/test/java/org/neo4j/unsafe/impl/batchimport/RelationshipGroupDefragmenterTest.java]: This can go away

**Comment by tinwelint** [comment]
@chrisvest a bit sloppy, removed those two lines

**Pull request merged**